### DISCUSSION TITLE
请问在Axis的label配置中，颜色存在偏差怎么解决？

### DISCUSSION THREAD

**Discussion by wangchao9240** [antvis/G2#5819] [Q&A]
### 问题描述

在官方给出的axis配置中 要配置label的颜色是需要labelFill这个属性

但我在实际使用以及官方示例中发现，即使颜色设成纯黑色#000 label的颜色也显示的是灰色

请问可以通过什么属性配置来实现label的无色差颜色显示吗？

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

### 屏幕截图或视频（可选）

![企业微信截图_6cfa43fa-fd91-4949-add5-7f92eaafb9af](https://github.com/antvis/G2/assets/21289186/9165a578-a773-4564-95b4-69f383123e9a)

**Comment by wangchao9240** [comment]
labelFillOpacity也设置成了1 但是图标的label还是显示的灰色

**Comment by pearmini** [comment]
> labelFillOpacity也设置成了1 但是图标的label还是显示的灰色

labelOpacity, labelStrokeOpacity 都设置成 1 试试？

**Comment by wangchao9240** [comment]
> > labelFillOpacity也设置成了1 但是图标的label还是显示的灰色
> 
> labelOpacity, labelStrokeOpacity 都设置成 1 试试？

也不行...还是灰色

**Comment by pearmini** [comment]
![image](https://github.com/antvis/G2/assets/49330279/9f5e8e96-dda7-45dc-b69d-db862d78d78f)

**Comment by chen476026986** [comment]
文档上写错了，不是labelFillOpacity,是labelOpacity，不是tickFillOpacity,而是tickOpacity，这两个设置为1，tickStroke和labelFill设置的颜色会有用
